src/test/shell/bazel/external_patching_test.sh

```diff
@@ -89,7 +91,7 @@ EOF
 -echo Here be dragons...
 +echo completely differently patched
 EOF
-  bazel build :foo.sh
+  bazel build :foo.sh | grep -q 'Hunk' || fail "expected verbose patch output"
```

Comment:
This construction does not work,
- on the one hand, as debug output comes on stderr, and,
- more importantly, is causes a broken pipe to the running bazel (as `grep -q` exits as soon as the string is found).